lib/src/generators/contract/openapi3-schema.ts

```diff
@@ -118,6 +142,26 @@ export interface OpenAPI3BaseSchemaType {
   };
 }
 
+export type OpenAPI3SchemaContentType =
+  | OpenAPI3SchemaApplicationJsonContentType
+  | OpenAPI3SchemaTextHtmlContentType
+
+export interface OpenAPI3SchemaApplicationJsonContentType extends OpenAPI3BaseSchemaType {
+  content: {
+    'application/json': {
+      schema: OpenAPI3SchemaType | null
+    }
+  }
+}
+
+export interface OpenAPI3SchemaTextHtmlContentType extends OpenAPI3BaseSchemaType {
+  content: {
+    'text/html': {
+      schema: OpenAPI3SchemaType | null
```

Comment:
When the content is `text/html`, isn't the schema always `string`?

Reply:
yes, that is correct. changing it